### ISSUE TITLE
Contradictory UI - Wallet password

### ISSUE THREAD

**Issue by davidraedev** [bisq-network/bisq#875]
When typing in a password, it allows you to type up to 50 chars, and when typing 51+ inline displays: The password you entered is too long. It cannot be **more than** 50 chars.

![screen shot 2017-07-20 at 12 34 44 am](https://user-images.githubusercontent.com/933763/28405969-6ae6561a-6ce3-11e7-9ba7-0c0ddc39759e.png)

But when you go to set password, if it is 50 chars, you get the error box saying that it has to be **less than** 50 chars.

![screen shot 2017-07-20 at 12 35 11 am](https://user-images.githubusercontent.com/933763/28405968-6ade721a-6ce3-11e7-8518-b33208e11a67.png)

So basically, the inline should be changed to say "more than 49 chars", and it should display at 50 chars, not 51 (along with greying out the set password button at 50 chars as well).

Seen on v0.5.3
